src/librustdoc/html/markdown.rs

```diff
@@ -715,6 +778,53 @@ impl LangString {
                         seen_other_tags = true;
                     }
                 }
+                x if extra.is_some() => {
+                    let s = x.to_lowercase();
```

Comment:
Without this line, the comparisons of `s == "compile_fail"` etc are pointless, but none of the test cases cover `COMPILE_FAIL` and friends.

Reply:
Arf indeed! Good catch!